WARNING 2023-12-20 18:29:51,277 log Not Found: /robots.txt
WARNING 2023-12-20 18:31:20,920 log Not Found: /sellers.json
WARNING 2023-12-20 18:31:20,922 log Not Found: /ads.txt
WARNING 2023-12-20 18:31:21,031 log Not Found: /app-ads.txt
WARNING 2023-12-20 18:31:21,392 log Not Found: /.well-known/assetlinks.json
WARNING 2023-12-20 18:31:21,393 log Not Found: /.well-known/apple-app-site-association
WARNING 2023-12-20 18:31:22,009 log Not Found: /static/basecamp/js/bootstrap.bundle.min.js.map
WARNING 2023-12-20 18:31:23,286 log Not Found: /robots.txt
WARNING 2023-12-20 18:31:23,520 log Not Found: /static/basecamp/css/sourcemaps/tiny-slider.css.map
WARNING 2023-12-20 18:31:23,521 log Not Found: /static/basecamp/css/theme.min.css.map
WARNING 2023-12-20 18:31:23,940 log Not Found: /.well-known/assetlinks.json
WARNING 2023-12-20 18:31:23,948 log Not Found: /.well-known/apple-app-site-association
WARNING 2023-12-20 18:31:23,951 log Not Found: /.well-known/gpc.json
WARNING 2023-12-20 18:31:23,954 log Not Found: /robots.txt
WARNING 2023-12-20 18:31:23,960 log Not Found: /.well-known/change-password
WARNING 2023-12-20 18:31:23,963 log Not Found: /.well-known/resource-that-should-not-exist-whose-status-code-should-not-be-200/
WARNING 2023-12-20 18:31:23,965 log Not Found: /.well-known/security.txt
WARNING 2023-12-20 18:31:50,841 log Not Found: /android-chrome-192x192.png
WARNING 2023-12-20 18:31:51,059 log Not Found: /robots.txt
WARNING 2023-12-20 18:31:51,264 log Not Found: /static/basecamp/js/jarallax.min.js.map
WARNING 2023-12-20 18:31:51,264 log Not Found: /static/basecamp/js/bootstrap.bundle.min.js.map
WARNING 2023-12-20 18:37:30,825 log Not Found: /robots.txt
WARNING 2023-12-20 18:40:28,653 log Not Found: /autodiscover/autodiscover.json
WARNING 2023-12-20 19:01:56,656 log Not Found: /contact/
ERROR 2023-12-20 19:03:34,210 log Internal Server Error: /booking_detail/
Traceback (most recent call last):
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/backends/sqlite3/base.py", line 328, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.IntegrityError: NOT NULL constraint failed: blog_post.company_name

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/basecamp/views.py", line 1121, in booking_detail
    p.save()
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/models/base.py", line 814, in save
    self.save_base(
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/models/base.py", line 877, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/models/base.py", line 1020, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/models/base.py", line 1061, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/horeb/github/easygo/venv/lib/python3.11/site-packages/django/db/backends/sqlite3/base.py", line 328, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.IntegrityError: NOT NULL constraint failed: blog_post.company_name
